### DISCUSSION TITLE
Are histograms implemented in the pandas backend?

### DISCUSSION THREAD

**Discussion by argenisleon** [ibis-project/ibis#4769] [Q&A] [answered]
Hi,

I am trying to calculate a column histogram like;

```
import pandas as pd
a=ibis.pandas.connect({"table": pd.DataFrame({"column_a": [1, 2, 3]})})
table = a.table("table")
table["column_a"].histogram(nbins=10).execute()
```

but I get 
`NotImplementedError: Could not find signature for execute_node: <Histogram, Series, int, NoneType, NoneType, NoneType>`

I tried with some param combinations without success. Any help?

**Comment by ogrisel** [comment, marked as answer]
I think the error is pretty explicit: this feature does not exist for this backend (yet).

If you use a recent enough version of ibis, you can use the duckdb backend on your Pandas dataframe via the `ibis.memtable` function:

```python
import ibis
import pandas as pd
table = ibis.memtable(pd.DataFrame({"a": range(100)}))
table["a"].histogram(nbins=10).execute()
```
